### DISCUSSION TITLE
K3s  -  memcache.go:121] couldn't get resource list

### DISCUSSION THREAD

**Discussion by vhuezo** [k3s-io/k3s#7431] [Q&A]
E0504 17:44:42.748706 2168494 memcache.go:287] couldn't get resource list for custom.metrics.k8s.io/v1beta1: the server is currently unable to handle the request
E0504 17:44:42.793466 2168494 memcache.go:121] couldn't get resource list for custom.metrics.k8s.io/v1beta1: the server is currently unable to handle the request
E0504 17:44:42.862365 2168494 memcache.go:121] couldn't get resource list for custom.metrics.k8s.io/v1beta1: the server is currently unable to handle the request
![2023-05-04_17-45](https://user-images.githubusercontent.com/41808201/236352843-dd80ecbc-0f89-4ab6-acdb-938a2e4510ed.png)

**Comment by brandond** [comment]
Figure out why your metrics-server pod is not running or is unreachable from the server node.

**Reply by ricardodani** [reply]
("Internal Server Error: \"/apis/metrics.k8s.io/v1beta1?timeout=32s\": the server could not find the requested resource") has prevented the request from succeeding
